packages/@aws-cdk/aws-ecs/test/cluster.test.ts

```diff
@@ -1024,6 +1024,20 @@ describe('cluster', () => {
     expect((cluster as any)._cfnCluster.serviceConnectDefaults.namespace).toBe('foo.com');
   });
 
+  test('allows setting cluster _defaultCloudMapNamespace for HTTP namespace', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = new ec2.Vpc(stack, 'MyVpc', {});
+    const cluster = new ecs.Cluster(stack, 'EcsCluster', { vpc });
+    // WHEN
+    const namespace = cluster.addDefaultCloudMapNamespace({
+      name: 'http://foo.com',
```

Comment:
This name is probably going to be invalid in CFN; I don't think we should validate but we should use an example in our test case which will be created successfully.

```suggestion
      name: 'foo',
```